tf/mod_source_parse_test.go

```diff
@@ -41,8 +41,9 @@ func TestParseGitSources(t *testing.T) {
 			source: "github.com/mineiros-io/example",
 			want: want{
 				parsed: tf.Source{
-					URL:  "https://github.com/mineiros-io/example.git",
-					Path: "github.com/mineiros-io/example",
+					URL:        "https://github.com/mineiros-io/example.git",
+					Path:       "github.com/mineiros-io/example",
+					PathScheme: "https",
```

Comment:
Specially looking at the tests it seems to me that PathScheme is actually just Scheme, in reference to the URL itself not Path. Sorry if Im missing something obvious here.

Reply:
After some discussions now I get that we need to model 2 schemes, so it is a little confusing in nature